django/db/models/functions/math.py

```diff
@@ -158,9 +158,13 @@ def get_group_by_cols(self, alias=None):
         return []
 
 
-class Round(Transform):
+class Round(FixDecimalInputMixin, NumericOutputFieldMixin, Transform):
     function = 'ROUND'
     lookup_name = 'round'
+    arity = None  # Override Transform's arity=1 to enable passing precision.
+
+    def __init__(self, expression, precision=0, **extra):
+        super().__init__(expression, precision, **extra)
```

Comment:
I think we should raise `ValueError` on SQLite when a negative precision is passed, e.g.:
```python
    def as_sqlite(self, compiler, connection, **extra_context):
        expression2 = self.get_source_expressions()[1]
        if isinstance(expression2, Value) and expression2.value < 0:
            raise ValueError('SQLite does not support negative precisions.')
        return super().as_sql(compiler, connection, **extra_context)
```
especially that it is simply treated as 0.

I would also remove admonition about _"negative values for precision argument"_.

Reply:
Because adding this would have required using `unittest.skipUnless()` checking `connection.vendor`, I changed to using a feature flag instead of adding to `django_test_skips` so that the approach was consistent and I could use both `skipIfDBFeature()` and `skipUnlessDBFeature()`.

I also changed this to use `precision` instead of `expression2` and it was necessary to `return super().as_sqlite(...)` to prevent breaking the transform -- the `RoundTests.test_transform` test case was failing.

Reply:
> Because adding this would have required using `unittest.skipUnless()` checking `connection.vendor`, I changed to using a feature flag instead of adding to `django_test_skips` so that the approach was consistent and I could use both `skipIfDBFeature()` and `skipUnlessDBFeature()`.

I think it's fine to use `django_test_skips` and `unittest.skipUnless()` checking `connection.vendor` (we did the same `NullIf` on Oracle). Implementation is SQLite-specific i.e. tests will crash on 3rd-party database backends even if they set a feature flag to `False`.

Reply:
Ok. Switched back.